engine/src/flutter/fml/logging.h

```diff
@@ -60,8 +60,26 @@ int GetVlogVerbosity();
 // kLogFatal and above is always true.
 bool ShouldCreateLogMessage(LogSeverity severity);
 
+constexpr bool ShouldCreateLogMessage2(LogSeverity severity, bool true_arg) {
```

Comment:
Include `Constexpr` in the name of these functions so it's clear why the wrappers exist.

Reply:
Done.